onnxruntime/core/framework/graph_partitioner.cc

```diff
@@ -818,6 +828,8 @@ Status GraphPartitioner::InlineFunctionsAOT(Model& model,
 
   model.RemoveLocalFunctionsProtos(not_inlined);
 
+  LOGS(logger, INFO) << "AOT inlining completed. Inlined local function definitions have been pruned.";
```

Comment:
Not part of this PR, but it might be useful to know how many functions got inlined. It might also give us some stats on how heavily functions are being used in newer models.